containers/src/Kokkos_StaticCrsGraph.hpp

```diff
@@ -299,43 +299,9 @@ class StaticCrsGraph {
   using row_block_type =
       View<const size_type*, array_layout, device_type, memory_traits>;
 
-  entries_type entries;
-  row_map_type row_map;
-  row_block_type row_block_offsets;
-
-  //! Construct an empty view.
-  KOKKOS_INLINE_FUNCTION
-  StaticCrsGraph() : entries(), row_map(), row_block_offsets() {}
-
-  //! Copy constructor (shallow copy).
-  KOKKOS_INLINE_FUNCTION
-  StaticCrsGraph(const StaticCrsGraph& rhs)
-      : entries(rhs.entries),
-        row_map(rhs.row_map),
-        row_block_offsets(rhs.row_block_offsets) {}
-
-  template <class EntriesType, class RowMapType>
-  KOKKOS_INLINE_FUNCTION StaticCrsGraph(const EntriesType& entries_,
-                                        const RowMapType& row_map_)
-      : entries(entries_), row_map(row_map_), row_block_offsets() {}
```

Comment:
I don't think it's worth it to remove this constructor and force users to change their code for a deprecated feature. If they have to change the namespace and use Kokkos Kernels anyway, it's more resonable to also change the API then.